src/auth/oauth/types.ts

```diff
@@ -4,7 +4,17 @@ export interface AuthQuery {
   timestamp: string;
   state: string;
   shop: string;
+  host?: string;
   hmac?: string;
+  shopify?: string[];
+}
+
+export interface LocalHmacQuery {
```

Comment:
Do we need the extra type here? We'll just ignore fields that we don't care about, so it wouldn't hurt to just use `AuthQuery` instead, would it?